src/skills-builder/skills-builder-modal/view-results/CarouselStack.jsx

```diff
@@ -45,7 +49,11 @@ const CarouselStack = ({ selectedRecommendations }) => {
         page: 'skills_builder',
         course_key: courseKey,
         product_type: productType,
-        selected_recommendations: selectedRecommendations,
+        selected_recommendations: {
+          job_id: jobId,
+          job_name: jobName,
+          course_keys: courseKeys,
+        },
```

Comment:
I wanted the fields here to be consistent with the rest of the event data so they have been `snake_cased`